requirements.txt

```diff
@@ -1 +1,2 @@
 neomodel
+urllib3
```

Comment:
python-requests should already include urllib3 and that is in the `scraper-requirements.txt` file.